lib/spack/spack/build_systems/racket.py

```diff
@@ -41,10 +42,10 @@ class RacketPackage(PackageBase):
     name = None  # type: Optional[str]
     parallel = True
 
-    @property
-    def homepage(self):
-        if self.pkgs:
-            return 'https://pkgs.racket-lang.org/package/{0}'.format(self.name)
+    @lang.ClassProperty
+    def _racket_homepage(cls):
```

Comment:
Where's this used?  I'm not seeing how `homepage` gets set now.

Reply:
Another typo that slipped in, thanks!

Reply:
Fixed in 4c073af41ca6